internal/db/schema/manager.go

```diff
@@ -243,85 +243,81 @@ func (b *Manager) ApplyMigrations(ctx context.Context) ([]RepairLog, error) {
 // runMigrations passes migration queries to a database driver and manages
 // the version and dirty bit. Cancellation or deadline/timeout is managed
 // through the passed in context.
-func (b *Manager) runMigrations(ctx context.Context, p *provider.Provider) ([]RepairLog, error) {
+func (b *Manager) runMigrations(ctx context.Context, p *provider.Provider) (retLogEntries []RepairLog, retErr error) {
 	const op = "schema.(Manager).runMigrations"
 
-	var logEntries []RepairLog
-	var errFinal error
-
 	if startErr := b.driver.StartRun(ctx); startErr != nil {
-		errFinal = errors.Wrap(ctx, startErr, op)
-		return nil, errFinal
+		retErr = errors.Wrap(ctx, startErr, op)
+		return
```

Comment:
There's no need to explicitly assign to `retErr`, you can keep this as just `err` as it was before Michael's PR. Or, just return the wrapped error directly
```suggestion
		return errors.Wrap(ctx, startErr, op)
```

Reply:
Also, please don't use bare returns. Even with named returns, be explicit